gene/fromGtf.go

```diff
@@ -105,8 +111,27 @@ func GtfToGene(g *gtf.Gene, ref []*fasta.Fasta) *Gene {
 					currCdsPos++
 				}
 			}
+
+			if val.ThreeUtr != nil {
+				for i = answer.startPos - (val.ThreeUtr.End - 1); i < answer.startPos-(val.ThreeUtr.Start-2); i++ {
+					answer.featureArray[i] = UtrThree
+				}
+				if answer.utrThree.start == 0 {
+					answer.utrThree.start = len(answer.cdnaSeq)
```

Comment:
Again, this seems like it could cause index-out-of-range problems; however, I might not be seeing the big picture.

Reply:
Same as above, this is relying on the fact that the cDNA is read in order.